src/locales/cy/pages.yml

```diff
@@ -1,9 +1,49 @@
 # title is managed in default.yml
+            
+landingPage:
+  title: Parhau i roi manylion eich cyfrif banc neu gymdeithas adeiladu
+  infoText1: Nesaf, byddwn yn gofyn am rywfaint o wybodaeth am eich cyfrif.
+  list1: 
+    list1Text: "Rhaid iddo fod:"
+    list1Bullet1: yn gyfrif cyfredol neu gyfrif cyfredol ar y cyd
+    list1Bullet2: gyda banc neu gymdeithas adeiladu yn y DU
+    list1Bullet3: yn eich enw chi
+  list2: 
+    list2Text: "Ni allwch ddefnyddio:"
+    list2Bullet1: cyfrif busnes
+    list2Bullet2: cyfrif cynilo, gan gynnwys Cyfrif Cynilo Unigol (ISA)
+  infoTitle: Beth i'w ddisgwyl
+  list3: 
+    list3Text: "Bydd angen i chi ddweud wrthym y:"
+    list3Bullet1: cod didoli
+    list3Bullet2: rhif cyfrif
+  infoText3: Byddwn yn gwirio enw eich cyfrif a'ch manylion gyda'ch banc neu gymdeithas adeiladu.
+  infoText4: Rhaid i'ch enw a manylion cyfrif gyfateb.
+  detailDropdown:
+    summaryText: Sut rydym yn defnyddio eich manylion banc
+    detailContent: <p class="instruction">Ni fyddwn yn cadw nac yn rhannu manylion eich cyfrif.</p><p class="instruction">Nid ydym yn defnyddio manylion eich cyfrif i wirio neu gymryd unrhyw daliadau.</p>
+  howContinueBankLanding: 
+    howContinueBankLink: /how-continue-bank
+    howContinueBankText: "Rwyf angen profi fy hunaniaeth mewn ffordd arall"
 
-nameEntry:
-  title: "Rhowch eich enw yn union fel y mae'n ymddangos ar eich ID gyda llun"
-  surname: "Enw olaf"
-  givenNames: "Enw olaf"
-  firstName: "Enw cyntaf"
-  middleName: "Enwau canol"
-  middleNameHint: "Gadewch hwn yn wag os nad oes gennych enwau canol"
\ No newline at end of file
+howContinueBank: 
+  title: Beth hoffech chi ei wneud?
+  warning: You cannot return to this screen if you look for another way to prove your identity.
```

Comment:
English?

Reply:
The translation for that element is part of a separate ticket (1607)

Reply:
Removed